### PR TITLE
An attempt at creating a simple, demo branching doc

### PR DESCRIPTION
**Pull request by paperclypse**
This seemed like a more useful solution than rewriting the style guide doc yet again.

### CODE REVIEW

File: src/install/demo/intro.mdx [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+headingText: Branching doc demo
+componentType: default
+---
+
+This is a demo branching doc. You can find the content for this doc at `src/install/demo` in our repo.
+
+Click the options to see how content is swapped in and out.
```
Comment (homelessbirds) [inline src/install/demo/intro.mdx:8]: @paperclypse - 

I think you could write something like:

"Below you will find a demo branching doc. You can find the content for this doc at `src/install/demo` in our repo.

As you explore the demo branching doc below, click the various options to see how content is swapped in and out."

File: src/install/demo/appInfo.mdx [hunk 2]
```diff
@@ -0,0 +1,38 @@
+---
+componentType: appInfoConfig
+headingText: Use appInfoConfig to show the dropdown options.
+descriptionText: 'Use the appInfoConfig componentType to show the dropdown options. You can also use "recommendedGuided: true" to display guided install or other content below the frontmatter section.'
```
Comment (homelessbirds) [inline src/install/demo/appInfo.mdx:4]: Do you think you could bold this text or make it italics or change it's color? My instinct is to make it stylistically stand out: "recommendedGuided: true"

File: src/install/demo/dropdown1/default.mdx [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+                <TabsBarItem id="powershell">Powershell</TabsBarItem>
+                <TabsBarItem id="web">Web</TabsBarItem>
+            </TabsBar>
+
+            <TabsPages>
+                <TabsPageItem id="curl">
+                Download the Java APM agent with this `curl` command and then unzip it.
+
+                ```sh
+                curl -0 https://download.newrelic.com/newrelic/java-agent/newrelic-agent/current/newrelic-java.zip
```
Comment (homelessbirds) [inline src/install/demo/dropdown1/default.mdx:30]: @paperclypse visually this doesn't make sense in the build. It shows up as a link below the tab example experience. Could you provide a little more context for this?

File: src/install/demo/dropdown1/default.mdx [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+            </TabsPages>
+            </Tabs>
+        ```
+
+    </TabsPageItem>
+    <TabsPageItem id="powershell">
+      Download the Java APM agent with this `curl` command and then unzip it.
+
+      ```
+      Invoke-WebRequest -Uri https://download.newrelic.com/newrelic/java-agent/newrelic-agent/current/newrelic-java.zip -OutFile newrelic-java.zip
```
Comment (homelessbirds) [inline src/install/demo/dropdown1/default.mdx:55]: Same as the above.

**Review by homelessbirds — DISMISSED** [review]
Great doc! I just had some minor questions and changes. Overall, it has my approval.

File: src/install/demo/intro.mdx (hunk 1 above)
Comment (paperclypse) [reply src/install/demo/intro.mdx:8]: Works for me!

File: src/install/demo/appInfo.mdx (hunk 2 above)
Comment (paperclypse) [reply src/install/demo/appInfo.mdx:4]: Sadly, I can't use Markdown formatting in these frontmatter fields.

File: src/install/demo/dropdown1/default.mdx (hunk 3 above)
Comment (paperclypse) [reply src/install/demo/dropdown1/default.mdx:30]: Yeah, I was trying to create a codeblock that showed the code for creating tabs, but I got lost in my own circular reasoning/code here :)

File: src/install/demo/dropdown1/default.mdx (hunk 4 above)
Comment (paperclypse) [reply src/install/demo/dropdown1/default.mdx:55]: Fixed it

**Pull request merged**